frontend/src/components/pages/touristicEvent/TouristicEventUI.tsx

```diff
@@ -74,8 +74,8 @@ export const TouristicEventUIWithoutContext: React.FC<Props> = ({
     // relative to the relative parent.
     scrollOffset:
       (sectionsContainerRef.current?.offsetTop ?? 0) -
-      sizes.desktopHeader -
-      sizes.detailsHeaderDesktop,
+      parseInt(theme.spacing.desktopHeader, 10) -
+      parseInt(theme.spacing[14], 10),
```

Comment:
```suggestion
      Number(theme.spacing.desktopHeader) -
      Numbertheme.spacing[14]),
```

Reply:
This won't work because the value is expressed in `px` like `230px`